src/diff/index.js

```diff
@@ -378,8 +393,21 @@ function catchErrorInComponent(error, component) {
 			}
 			catch (e) {
 				error = e;
+				isSuspend = typeof error.then === 'function';
```

Comment:
When will this `catch(e)` handle a suspend? I don't think any of the above code will ever invoke a synchronous render (`setState`, `enqueueRender`, etc. all trigger an async rerender), so the code this `catch` watches should never throw a promise.

If that's true then I think you can remove the `isSuspend` and `suspendingComponent` lines from this `catch`.

Reply:
Done, saved 5 bytes 🎉 . 

I kept `isSuspend = false` in the catch tough. Dropping this would save another 3B but errors thrown in `_childDidSuspend` would result in a `Missing Suspense` error. What do you think? Remove the `isSuspend = false` too?